app/models/action.rb

```diff
@@ -0,0 +1,5 @@
+class Action < Sequel::Model
```

Comment:
Subclasses of Sequel::Model are mapped to rows from the database table whose name matches ('actions' for Action).
There are getters and setters automatically defined for each column.